Patch to include unitstd.h in the parameters for cpp
Thekla Loizou July 2015
--- param.cpp.orig	2014-12-08 22:13:01.439639000 +0200
+++ param.cpp	2014-12-08 22:14:20.305028000 +0200
@@ -1,3 +1,4 @@
+#include <unistd.h>
 #include "param.h"
 #include<iostream>
 
